docs/src/lib/MDX_COMPONENTS.js

```diff
@@ -10,17 +10,17 @@ import {
   AreaPlaceholder,
   Boundaries,
   Button,
-  ChatMessages,
```

Comment:
I'm confused, why is this renaming of components happening in this PR?

Reply:
This is not correct. I am investigating, I did not touch this file.

Reply:
The PR was pointing to `main`, which is incorrect. Branch revisited and correctly set. 👍